src/model.jl

```diff
@@ -542,28 +542,26 @@ function _add_profile_generator_limits!(
     JuMP.@constraint(
         m,
         profile_upper_bound[
-            g in keys(sets.profile_resources_num_rep),
-            i in 1:length(sets.profile_resources_idx[sets.profile_resources_num_rep[g]]),
+            g in values(sets.profile_resources_num_rep),
```

Comment:
`JuMP` is okay with using strings as indices instead of integers? If so, thanks for catching that! My original formulation is a lot grosser because I thought you needed to use integers to index the constraints.